src/routes/github/manifest/github-manifest-complete-get.ts

```diff
@@ -0,0 +1,36 @@
+import { Request, Response } from "express";
+import axios from "axios";
+import { GitHubServerApp } from "~/src/models/github-server-app";
+import { Installation } from "~/src/models/installation";
+
+export const GithubManifestCompleteGet = async (req: Request, res: Response) => {
+	const uuid = req.params.uuid;
```

Comment:
where is this coming from?

Reply:
maybe we can put both uuid and gheHost in the session cookie, as sessio cookie is signed cannot be tempered.